target/linux/lantiq/base-files/lib/functions/lantiq_dsl.sh

```diff
@@ -472,12 +471,14 @@ errors() {
 	local fecf
 
 	lsctg=$(dsl_cmd pmlsctg 1)
+	fecsf=$(dsl_val "$lsctg" nFECS)
```

Comment:
You should define fecsf as a local variable and remove fecf.

Reply:
You are right. Will do that on monday.